cmd/access/node_builder/unstaked_access_node_builder.go

```diff
@@ -46,15 +51,44 @@ func (builder *UnstakedAccessNodeBuilder) Initialize() cmd.NodeBuilder {
 
 func (builder *UnstakedAccessNodeBuilder) validateParams() {
 
-	// for an unstaked access node, the staked access node ID must be provided
-	if strings.TrimSpace(builder.stakedAccessNodeIDHex) == "" {
-		builder.Logger.Fatal().Msg("staked access node ID not specified")
-	}
-
-	// and also the unstaked bind address
+	// for an unstaked access node, the unstaked network bind address must be provided
 	if builder.unstakedNetworkBindAddr == cmd.NotSet {
 		builder.Logger.Fatal().Msg("unstaked bind address not set")
 	}
+
+	if len(builder.bootstrapNodeAddresses) != len(builder.bootstrapNodePublicKeys) {
+		builder.Logger.Fatal().Msg("number of bootstrap node addresses and public keys should match")
+	}
+}
+
+// deriveBootstrapPeerIdentities derives the Flow Identity of the bootstreap peers from the parameters.
+// These are the identity of the staked and unstaked AN also acting as the DHT bootstrap server
+func (builder *UnstakedAccessNodeBuilder) deriveBootstrapPeerIdentities() {
+
+	builder.bootstrapIdentites = make([]*flow.Identity, len(builder.bootstrapNodeAddresses))
+	for i, address := range builder.bootstrapNodeAddresses {
+		key := builder.bootstrapNodePublicKeys[i]
+
+		// json unmarshaller needs a quotes before and after the string
+		// the pflags.StringSliceVar does not retain quotes for the command line arg even if escaped with \"
+		// hence this additional check to ensure the key is indeed quoted
+		if !strings.HasPrefix(key, "\"") {
+			key = fmt.Sprintf("\"%s\"", key)
+		}
+		// networking public key
+		var networkKey encodable.NetworkPubKey
+		err := json.Unmarshal([]byte(key), &networkKey)
+		builder.MustNot(err)
+
+		// create the identity of the peer by setting only the relevant fields
+		id := &flow.Identity{
+			NodeID:        flow.ZeroID, // the NodeID is the hash of the staking key and for the unstaked network it does not apply
```

Comment:
Till we figure out [5739](https://github.com/dapperlabs/flow-go/issues/5739), setting the ID to ZeroID